app/services/invoices/create_advance_charges_invoice_subscription_service.rb

```diff
@@ -2,7 +2,7 @@
 
 module Invoices
   class CreateAdvanceChargesInvoiceSubscriptionService < BaseService
-    Result = BaseResult
+    Result = EmptyResult
```

Comment:
Actually, this is the same as
```suggestion
    Result = BaseResult
```

Reply:
This is absolutely exactly the same 🤦‍♂️ Thank you!